apps/test/unit/templates/sectionsRefresh/AdvancedSettingTogglesTest.jsx

```diff
@@ -1,101 +1,74 @@
         updateSection={() => {}}
         section={{pairingAllowed: true, restrictSection: false}}
       />
     );
-    expect(
-      wrapper.find('ToggleSwitch[id="uitest-pair-toggle"]').props().isToggledOn
-    ).to.be.true;
-    expect(
-      wrapper.find('ToggleSwitch[id="uitest-lock-toggle"]').props().isToggledOn
-    ).to.be.false;
+    const pairingToggle = screen.getByLabelText(i18n.pairProgramming());
+    expect(pairingToggle).toHaveAttribute('checked');
+    const lockToggle = screen.getByLabelText(i18n.restrictSectionAccess());
+    expect(lockToggle).not.toHaveAttribute('checked');
+    // Check that the other toggles do not appear.
+    const ttsToggle = screen.queryByText(i18n.enableTtsAutoplayToggle());
+    expect(ttsToggle).toBeNull();
+    const lessonExtrasToggle = screen.queryByText(
+      i18n.enableLessonExtrasToggle()
+    );
+    expect(lessonExtrasToggle).toBeNull();
+    const aiTutorToggle = screen.queryByText(i18n.enableAITutor());
+    expect(aiTutorToggle).toBeNull();
   });
 
-  it('renders Lesson Extras Toggle when available', () => {
-    const wrapper = shallow(
+  it('renders Lesson Extras toggle when available, enabled', () => {
+    render(
       <AdvancedSettingToggles
         updateSection={() => {}}
         section={{
-          pairingAllowed: true,
-          restrictSection: false,
-          ttsAutoplayEnabled: false,
           lessonExtras: true,
         }}
         hasLessonExtras={true}
       />
     );
-    expect(
-      wrapper.find('ToggleSwitch[id="uitest-lesson-extras-toggle"]')
-    ).to.have.lengthOf(1);
-  });
-
-  it('renders TTS Toggle when available', () => {
-    const wrapper = shallow(
-      <AdvancedSettingToggles
-        updateSection={() => {}}
-        section={{
-          pairingAllowed: true,
-          restrictSection: false,
-          ttsAutoplayEnabled: false,
-          lessonExtras: true,
-        }}
-        hasTextToSpeech={true}
-      />
+    const lessonExtrasToggle = screen.getByLabelText(
+      i18n.enableLessonExtrasToggle()
     );
-    expect(
-      wrapper.find('ToggleSwitch[id="uitest-tts-toggle"]')
-    ).to.have.lengthOf(1);
+    expect(lessonExtrasToggle).toHaveAttribute('checked');
   });
 
-  it('renders TTS Toggle when available', () => {
-    const wrapper = shallow(
+  it('renders TTS toggle when available, disabled', () => {
+    render(
       <AdvancedSettingToggles
         updateSection={() => {}}
         section={{
-          pairingAllowed: true,
-          restrictSection: false,
           ttsAutoplayEnabled: false,
-          lessonExtras: true,
         }}
         hasTextToSpeech={true}
       />
     );
-    expect(
-      wrapper.find('ToggleSwitch[id="uitest-tts-toggle"]')
-    ).to.have.lengthOf(1);
+    const ttsToggle = screen.getByLabelText(i18n.enableTtsAutoplayToggle());
+    expect(ttsToggle).not.toHaveAttribute('checked');
   });
 
-  it('changes the status of pair programming setting when clicked', () => {
```

Comment:
I removed this test case because we already have coverage over the functionality that a click will change the state of the the toggle in the [test](https://github.com/code-dot-org/code-dot-org/blob/c0eebfaf013908be88c00ed75c27484db5a01da3/apps/test/unit/componentLibrary/ToggleTest.jsx#L19) for the `Toggle` component.